package.json

```diff
@@ -132,7 +132,7 @@
     "@hapi/podium": "^4.1.3",
     "@hapi/vision": "^6.1.0",
     "@hapi/wreck": "^17.1.0",
-    "@opensearch-project/opensearch": "^1.1.0",
+    "@opensearch-project/opensearch": "^2.1.0",
```

Comment:
Do we know if the breaking changes are mitigated in OSD?

Reply:
I am now aware of breaking change. Can you elaborate more? According to this, `opensearch-project/opensearch 2.1.0` seems to support both OpenSearch `1.x` and `2.0` https://github.com/opensearch-project/opensearch-js/issues/303#issuecomment-1267456579